src/shell/lisa_shell

```diff
@@ -93,13 +93,27 @@ fi
 echo -ne "$LISASHELL_DEFAULT"
 }
 
+function _lisa-update-all {
+echo -ne "${LISASHELL_BLUE}"
+echo -e "*** Update LISA installation"
+
+git pull
```

Comment:
I think that solution it's sound and reasonably safe at the same time ;-)
Most of the users will probably always be on master... while, it they get a bice and bold
   WARNING: you have uncommitted changes
or
  WARNING: you are not on master branch
they will know what is required before proceeding with the update.